modules/rhel-compute-updating.adoc

```diff
@@ -53,10 +51,10 @@ By default, the base OS RHEL with "Minimal" installation option enables firewall
 +
 [source,terminal]
 ----
-# subscription-manager repos --disable=rhel-7-server-ose-4.8-rpms \
-                             --enable=rhel-7-server-ansible-2.9-rpms \
-                             --enable=rhel-7-server-ose-4.9-rpms
+# subscription-manager repos --disable=rhel-7-server-ose-4.9-rpms \
+                             --enable=rhel-7-server-ose-4.10-rpms
 ----
+// As of OCP 4.10 GA, there are no openshift-ansible el8 packages built for RHEL8, so QE advised that we still run the playbook on RHEL7 machines. 
 
 .. On the machine that you run the Ansible playbooks, update the required packages, including `openshift-ansible`:
```

Comment:
@codyhoag I'm not sure if we want to emphasize this machine is a RHEL-7 host, to avoid confusing on the different repo names for the following operations.

Reply:
Thanks for the confirmation, it's ok for me.